examples/tensorflow/real-time-object-detection-rpi4/README.md

```diff
@@ -0,0 +1,39 @@
+# Real-time Object Detection on Raspberry Pi 4
+
+## Requirements
+
+- Raspberry Pi 4 with Raspberry Pi OS (64-bit)
+- Python 3.11
+
+## Dependencies
+
+Before running the code, make sure to install the following dependencies:
+
+```bash
+sudo rm /usr/lib/python3.11/EXTERNALLY-MANAGED
+sudo pip3 install opencv-python==4.8.1.78
+sudo pip3 install mediapipe==0.10.8
```

Comment:
I'm a bit hesitant about advising people to revert Bookworm's default behaviour which makes folks use venvs. Is it possible to do this in a venv?

Reply:
Hello @davidplowman , I appreciate your concern about Bookworm's default behavior, and I understand the importance of encouraging the use of virtual environments (venvs). You can certainly perform the installation within a venv. Here's an updated set of instructions:

### Create and activate a virtual environment
```bash
python3.11 -m venv myenv
source myenv/bin/activate
```

### Install required packages within the virtual environment
```bash
pip install opencv-python==4.8.1.78
pip install mediapipe==0.10.8
```

<blockquote> 
<p dir="auto">Your subsequent commands and steps can be executed within the venv.</p>
</blockquote> 

<blockquote> 
<p dir="auto">Don't forget to deactivate the venv when you're done.</p>
</blockquote> 

```bash
deactivate
```

Reply:
I would urge caution around venvs.

I have had trouble running https://github.com/raspberrypi/picamera2/blob/main/examples/tensorflow/real_time.py because:
* With system python (outside of a venv) I can't install `tflite-runtime`
* In a venv PyQt/OpenCV is problematic - PyQt5 is not installable (`pip install PyQt5` on RPi 5 4GB results in OOM crash), so preview windows don't work and opencv imshow is badly behaved (usually fails to display)
* In a conda enviroment it is possible to run `real_time.py`, but only with a very heavy PyQt5 / OpenCV install, and the example doesn't seem complete (perhaps a README would help)

Ideally it would be possible to run all of the examples in a python virtualenv, but realistically I don't see how that's possible given the reliance on PyQt for preview windows.